source/principles/secure-by-design-principles/index.html.md.erb

```diff
@@ -0,0 +1,23 @@
+---
+title: Secure by Design principles
+weight: 30
+---
+
+# <%= current_page.data.title %>
+
+[Secure by Design (SbD)](https://www.security.gov.uk/policy-and-guidance/secure-by-design/) is a cross-government approach that will be introduced to the Department for Education (DfE) in January 2025. It sets out what delivery teams and security professionals need to do to incorporate effective cyber security practices when building digital services and technical infrastructure.
+
+From January, any project delivering changes to digital services or technology infrastructure that is in scope of the [digital and technology spend controls approval process](https://www.gov.uk/government/publications/digital-and-technology-spend-control-version-6) must follow [Secure by Design principles](https://www.security.gov.uk/policy-and-guidance/secure-by-design/principles/). Digital services which are in operation or undergoing routine maintenance are not in scope for January. Over time, all projects will be required to follow Secure by Design principles and this will be agreed as part of the DfE CIS Division strategy and roadmap for SbD.
```

Comment:
Secure by Design can be shortened to 'SbD' throughout this paragraph here as we've used it in line 8.
Do we think we need to spell out CIS for this? I'm not sure if internal people will know off the bat.